config/m4/tl_coll_plugins.m4

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+AC_DEFUN([CHECK_TL_COLL_PLUGINS],[
+    AC_ARG_WITH([tlcp],
+            [AS_HELP_STRING([--with-tlcp=(tl1,tl2)], [Enable build of TL collectives plugins])],
```

Comment:
how this tl1,tl2 filtering works?

Reply:
right, long ago when i opened a PR i wanted to add it but never did. Now, i've added the logic.